README.md

```diff
@@ -12,14 +12,66 @@ Storj is in the midst of a rearchitecture. Please stay tuned for our v3 whitepap
 
 Storj is a platform, token, and suite of decentralized applications that allows you to store data in a secure and decentralized manner. Your files are encrypted, shredded into little pieces and stored in a global decentralized network of computers. Luckily, we also support allowing you (and only you) to recover them!
 
-## To start developing
+# Start Using Storj
+
+### Download the latest release
+
+Go here to download the latest build
+
+### Configure AWS CLI
+
+In a new terminal session:
+
+Download and install the AWS CLI: https://docs.aws.amazon.com/cli/latest/userguide/installing.html
```

Comment:
you should add the command i have, if pip and install doesn't work, with the -H flag. This will save a lot of people time. Or make a note "if this doesn't work, use this command to download the aws cli"

Reply:
I feel like we shouldn't worry about this; AWS and pip should have enough documentation online to help users through an issue like this. I don't know what the appropriate level of help for installing a service that's not ours is.

Reply:
ohh hmm. yeah , it can go either way. It took me a minute to get it installed when pip wasn't working. It seems there is some incompatibilities with the installation and some computers.  I just thought it would be helpful to have it.

Reply:
the aws installation website has a zip file i think you can just download and run that doesn't require pip